### DISCUSSION TITLE
Example of Drug Measurement and Unit of Measure

### DISCUSSION THREAD

**Discussion by TCL2023** [CMSgov/hospital-price-transparency#47] [Q&A] [answered]
Can you update the CSV wide format with an example for Morphine Sulfate?

The item description from the hospital states Morphine Sulfate PCA 250MG. But the HCPC description is for J2270 injection morphine sulfate, up to 10mg.  
Would the unit of measure be1 and the drug measurement be MG?   

Since this example is for 250mg (more than 10mg per HCPC).  Would there be a column under the code header indicating the charge is set with a multiplier of 25?

I have done a search and noted a similar question that has gone unanswered  since January 2, 2024.

**Comment by RidenourK** [comment, marked as answer]
@TCL2023 Thank you for your question and your patience.

Please refer to rows 15-21of the CSV Wide examples, rows 19-30 of the CSV Tall examples, and lines 197-444 of the JSON examples available on this repository for [examples](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples) of how you could encode drug unit and drug type of measurement.
